src/ws/login.html

```diff
@@ -35,26 +35,26 @@
                 <label id="conversation-prompt" for="conversation-input"></label>
               </div>
               <div class="col-sm-12 col-md-12">
-                <input type="password" class="form-control" id="conversation-input">
+                <input type="password" class="form-control" id="conversation-input" />
               </div>
             </div>
 
             <div id="user-group" class="form-group">
               <label for="login-user-input" class="col-sm-2 col-md-2 control-label" translate>User name</label>
               <div class="col-sm-10 col-md-10">
-                <input type="text" class="form-control" id="login-user-input" autocorrect="off" autocapitalize="none" autofocus>
+                <input type="text" class="form-control" id="login-user-input" autocorrect="off" autocapitalize="none" autofocus />
               </div>
             </div>
 
             <div id="password-group" class="form-group">
               <label for="login-password-input" class="col-sm-2 col-md-2 control-label" translate>Password</label>
               <div class="col-sm-10 col-md-10">
-                <input type="password" class="form-control" id="login-password-input">
+                <input type="password" class="form-control" id="login-password-input" />
```

Comment:
The "/" is optional, and we don't use it elsewhere, right?  I think we shouldn't add it here.

Reply:
It's optional for HTML 5 documents, as that spec defines it as a (void element)[https://www.w3.org/TR/html5/syntax.html#void-elements].

We're pretty inconsistent in Cockpit wit regards to putting the closing `/` in void elements. I haven't counted, but seen a lot of invocations of both. I don't think changing from one to the other helps, unless we'd make all of them the same. Could you please remove those again?

Reply:
It appals my XHTML stylistic sense ☺ but ok, reverted that bit.